include/zmq.h

```diff
@@ -670,6 +670,12 @@ ZMQ_EXPORT void zmq_threadclose (void *thread_);
 #define ZMQ_SOCKS_PASSWORD 100
 #define ZMQ_IN_BATCH_SIZE 101
 #define ZMQ_OUT_BATCH_SIZE 102
+#define ZMQ_TLS_KEY_PEM 103
```

Comment:
if these are only for WS, perhaps it's worth naming ZMQ_WS_...? To avoid confusion

Reply:
I will change it (to WSS), actually adding TLS to zmtp will be simple now and would make the name relevant, but if we do we can change them then and obsolete the previous names.

Reply:
fixed